client/questionApi.js

```diff
@@ -0,0 +1,16 @@
+export const questionApi = {
```

Comment:
Bra å splitte ut API i egen fil og kode! Men dere bør kanskje lage en `fetchJSON` hjelpemetode. (For kun et API som her, er det ikke så farlig, da)